Tests/UnitTests/Reporters/JunitXMLReporterTest.jsl

```diff
@@ -140,20 +141,19 @@ ut test(JunitXMLReporters, "Counts are correct at root", Expr(
 // 9.0 JunitXMLReporter one simple passing
 
 ut test( JunitXMLReporters, "JunitXMLReporter one simple passing", Expr(
-
+//include("../../../Source/Reporters/JunitXMLReporter.jsl");
```

Comment:
Remove the dead code please. I hope that the issue you were trying to solve with this code are be addressed by #66 and #67. Try building and installing the JSL-Hamcrest-Dev addin if you haven't already. See [README.md#releasing-a-version](https://github.com/sassoftware/jsl-hamcrest/blob/master/README.md#releasing-a-version) for details.

Reply:
done in de325ccbad38866829e3e311823355d232c5b9fd